plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerLabelProvider.java

```diff
@@ -204,7 +204,14 @@ public Object visit(ITags capability) {
 				return NLS.bind("({0})", org.apache.commons.lang.StringUtils.join(capability.getTags(), ", "));
 			}
 		}, null);
-		return style(template.getName(), tags);
+		StringBuilder qualifier = new StringBuilder();
+		if(!StringUtils.isEmpty(tags)) {
```

Comment:
Sorry, I am late to reply.

There is comparable number of usages in openshift of org.apache.commons.lang.StringUtils.isBlank(+isEmpty+isNotBlank+isNotEmpty) and org.jboss.tools.openshift.common.core.utils.StringUtils.isEmpty (the last one takes some lead). I would not mind to change to apache commons since it is already used in this file.

By the way, this file contains for the same property two different calls:
- org.apache.commons.lang.StringUtils.isNotEmpty(project.getDisplayName()) in getText(Object)
- org.apache.commons.lang.StringUtils.isNotBlank(project.getDisplayName()) in getStyledText(IProject)
Should we revise usage of these two methods to make it consistent?

Also, maybe we may implement org.jboss.tools.openshift.common.core.utils.StringUtils.isEmpty with org.apache.commons.lang.StringUtils.isEmpty to make it formally of no matter which one to use. 

@jcantrill, @adietish, what do you think?